cookbook/images/fading-in-images.md

```diff
@@ -68,3 +68,61 @@ class MyApp extends StatelessWidget {
 ```
 
 ![Fading In Image Demo](/images/cookbook/fading-in-images.gif)
+
+### From Asset Bundle
+
+You can also consider using local assets for placeholders. First, add the asset
+to the project’s _pubspec.yaml_ file:
+
+<!-- skip -->
+```diff
+ flutter:
+   assets:
++    - assets/loading.gif
+```
+
+(See [Assets and Images](/assets-and-images/) for a full guide to adding assets.)
+
+Then, use the [`FadeInImage.assetNetwork`](https://docs.flutter.io/flutter/widgets/FadeInImage/FadeInImage.assetNetwork.html)
+constructor:
+
+<!-- skip -->
+```dart
+FadeInImage.assetNetwork(
+  placeholder: 'assets/loading.gif',
+  image: 'https://github.com/flutter/website/blob/master/_includes/code/layout/lakes/images/lake.jpg?raw=true',
+);
+```
+
+### Complete Example
+
+```dart
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(new MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final title = 'Fade in images';
+
+    return new MaterialApp(
+      title: title,
+      home: new Scaffold(
+        appBar: new AppBar(
+          title: new Text(title),
+        ),
+        body: Center(
+          child: new FadeInImage.memoryNetwork(
```

Comment:
Hey there, thanks for the contribution!

It looks like this is the line that's causing the build to fail. I think if ya change this line to `FadeInImage.assetNetwork(...`, you should be all good!

Reply:
Oh, whoops! Yep, I can do that.

Reply:
Fixed in 38a1491a5ae846944c68daf020b33815a8361c2f.